Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM centos:7
+
+RUN yum install centos-release-scl -y \
+    && yum-config-manager --enable centos-sclo-rh-testing \
+    && yum install -y \
+        rh-postgresql96 \
+        rh-postgresql96-postgresql-devel \
+        rh-python36 \
+        rh-python36-python-pip \
+        libffi-devel \
+        gcc
+
+WORKDIR /srv
+
+RUN scl enable rh-python36 -- \
+    pip install pipenv
+
+RUN scl enable rh-python36 -- \
+    pipenv install --dev
+
+COPY saltcellar .
+
+ENTRYPOINT ["scl", "enable", "rh-postgresql96", "rh-python36", "--", "gunicorn"]
+CMD ["-c","config/gunicorn.py","config.wsgi"]
```

Comment:
Maybe I'm jumping the gun and this is in the works, but where's config/gunicorn.py?